tests/Integration/Listeners/TriggerAlertIfConditionsPassListenerTest.php

```diff
@@ -256,11 +256,6 @@ public function testHandleCreateAlertBasedOnTag()
             'comparator' => Condition::COMPARATOR_LIKE,
             'value' => 'bill'
         ]);
-        $alert->conditionals()->create([
-            'parameter' => 'tag.name.en',
-            'comparator' => Condition::COMPARATOR_LIKE,
-            'value' => 'subscription'
-        ]);
```

Comment:
SMH. This test exploited https://github.com/austinkregel/finance/pull/13/files#r448101974
It's like it was an unintentionally intentional feature....